src/types/lists.ts

```diff
@@ -22,3 +22,12 @@ export type UNSDG_NUMS =
 
 export type TransactionStatus = "open" | "approved" | "expired";
 export type EndowmentType = "charity" | "ast" | "daf";
+
+//https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types
+type ImageSubType = "svg" | "jpeg" | "png" | "webp";
+type ApplicationSubType = "pdf";
```

Comment:
@ap-justin 
Nit: We have "image" subtype. Maybe call the other generic files "documents" or "documentation" subtypes instead of a naming based on the place it's used in? We could have this support files like doc,xlsx,tiff,etc in the future and use it in more places. It would align will with the eventual "video" subtypes needed by Media. 
```suggestion
type DocumentationSubType = "pdf";
```

Reply:
the keyword `application` is one of the [discreet types](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types#application) where it's subtypes are `pdf, zip, pkc8 ...`, 

other formats like `doc` and `docx ` is also a subtype of `application`
<img width="1158" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/d14ef4a9-bc56-455c-818a-f48ddb36ff25">

Reply:
Ah! :bulb: _That_ Application. Got it! All good then. :)